Source/JavaScriptCore/bytecode/ExpressionInfo.h

```diff
@@ -42,7 +43,7 @@ namespace JSC {
 
 class ExpressionInfo {
     WTF_MAKE_NONCOPYABLE(ExpressionInfo);
-    WTF_DEPRECATED_MAKE_FAST_ALLOCATED(ExpressionInfo);
+    WTF_MAKE_TZONE_ALLOCATED(ExpressionInfo);
```

Comment:
This is wrong.  `ExpressionInfo` is dynamically sized and cannot be TZone'd.  See `ExpressionInfo::Encoder::createExpressionInfo()`.